SwiftVoiceCallKitQuickstart/ViewController.swift

```diff
@@ -342,7 +348,7 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
         // TwilioVoice.configureAudioSession()
         
         assert(action.callUUID == self.callInvite?.uuid)
-        
+        TwilioVoice.isAudioEnabled = false
```

Comment:
Should we localize this code to the reportNewIncomingCallWithUUID block where we actually call `TwilioVoice.configureAudioSession()?